### PR TITLE
[meteor/eslint-plugin-meteor] chore: remove unused scripts

### PR DESCRIPTION
**Pull request by afrokick**
- I found an issue  in https://github.com/meteor/meteor/commit/bca245cd  Seems like we don't need the `scripts` folder. So I remove it. ( @StorytellerCZ maybe you remember why it contains in the commit? or is it just by a mistake? )
- replace readme by original from https://github.com/meteor/eslint-plugin-meteor#readme

### CODE REVIEW

**Comment by StorytellerCZ** [comment]
Hi @afrokick, I don't think I did much of a check there and just copied over the original code, so I can't comment beyond that. I would be cautious about removing these. Some of them maybe tests that we haven't yet re-implemented in the CI and some look like some scripts for publication, but I can't comment beyond that as I don't remember.

**Comment by StorytellerCZ** [comment]
Hi @afrokick, I don't think I did much of a check there and just copied over the original code, so I can't comment beyond that. I would be cautious about removing these. Some of them maybe tests that we haven't yet re-implemented in the CI and some look like some scripts for publication, but I can't comment beyond that as I don't remember.

**Comment by Grubba27** [comment]
Hello @afrokick, the script folder contains many utilities for the maintainers so that we can publish easier Meteor versions.
Such as creating the tarballs that will be the ./meteor that we use to run our apps. A way to automate the bump in the semver for the packages and many others

**Comment by afrokick** [comment]
@Grubba27 you mean `root/scripts`? But this PR not about it, but `root/npm-packages/eslint-plugin-meteor/scripts`

**Comment by Grubba27** [comment]
Wow. I read it and thought it was the root ones, but you are correct. I'm confused why there are those files there

**Comment by afrokick** [comment]
@Grubba27 you can remove it locally to test it for safety.

**Comment by afrokick** [comment]
Ok, I merged the dev branch.

File: scripts/dev-bundle-tool-package.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ var packageJson = {
     pacote: "https://github.com/meteor/pacote/tarball/a81b0324686e85d22c7688c47629d4009000e8b8",
     "node-gyp": "8.0.0",
     "node-pre-gyp": "0.15.0",
-    typescript: "4.5.4",
+    typescript: "4.6.4",
```
Comment (afrokick) [inline scripts/dev-bundle-tool-package.js:17]: @Grubba27 I updated to properly version which we use in meteor-babel

**Pull request closed without merging**